### PR TITLE
Power config

### PR DESCRIPTION
**Pull request by Nicolai-Bak**

### CODE REVIEW

File: Microwave.Test.Unit/UserInterfaceTest.cs [hunk 1]
```diff
@@ -83,17 +113,17 @@ public void Ready_2PowerButton_PowerIs100()
         [Test]
         public void Ready_14PowerButton_PowerIs700()
         {
-            for (int i = 1; i <= 14; i++)
+            for (int i = 1; i <= 16; i++)
```
Comment (Peter-Trang) [inline Microwave.Test.Unit/UserInterfaceTest.cs:116]: why has the value of "i" been changed from 14 to 16?

File: Microwave.Test.Unit/UserInterfaceTest.cs [hunk 2]
```diff
@@ -83,17 +113,17 @@ public void Ready_2PowerButton_PowerIs100()
...
             {
                 powerButton.Pressed += Raise.EventWith(this, EventArgs.Empty);
             }
-            display.Received(1).ShowPower(Arg.Is<int>(700));
+            display.Received(1).ShowPower(Arg.Is<int>(800));
         }
 
         [Test]
-        public void Ready_15PowerButton_PowerIs50Again()
+        public void Ready_17PowerButton_PowerIs50Again()
```
Comment (Peter-Trang) [inline Microwave.Test.Unit/UserInterfaceTest.cs:124]: Why the name change from Ready_15 to Ready_17?

File: Microwave.Test.Unit/UserInterfaceTest.cs (hunk 1 above)
Comment (Peter-Trang) [inline Microwave.Test.Unit/UserInterfaceTest.cs:116]: why has the value of "i" been changed from 14 to 16?

Comment (Nicolai-Bak) [reply Microwave.Test.Unit/UserInterfaceTest.cs:116]: fordi maxpower for uut er ændret fra 700 til 800 for at vise at dette nu sættes i construktoren

File: Microwave.Test.Unit/UserInterfaceTest.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (Nicolai-Bak) [reply Microwave.Test.Unit/UserInterfaceTest.cs:124]: fordi maxpower for uut er ændret fra 700 til 800 for at vise at dette nu sættes i construktoren

**Review by Peter-Trang — APPROVED** [review]

**Pull request merged**